core/lib/dal/src/models/tests.rs

```diff
@@ -0,0 +1,356 @@
+use chrono::Utc;
+use zksync_types::{
+    fee::Fee,
+    l1::{OpProcessingType, PriorityQueueType},
+    l2::TransactionType,
+    Address, Bytes, Execute, ExecuteTransactionCommon, Transaction, EIP_1559_TX_TYPE,
+    EIP_2930_TX_TYPE, EIP_712_TX_TYPE, H160, H256, PRIORITY_OPERATION_L2_TX_TYPE,
+    PROTOCOL_UPGRADE_TX_TYPE, U256,
+};
+use zksync_utils::bigdecimal_to_u256;
+
+use crate::{models::storage_transaction::StorageTransaction, BigDecimal};
+
+/// Creates an "empty" transaction with all values set to None / zeros / left empty etc.
+///
+/// Used by other tests as a template to create transactions of a specific type.
+fn empty_tx() -> StorageTransaction {
+    StorageTransaction {
+        gas_limit: None,
```

Comment:
I suspect that you can just return `StorageTransaction::default()` here, it should be the same.

Reply:
There seem to be no `default` method there - it doesn't implement `Default` trait

Reply:
I would suggest to add such a derive and it can even be gated under `#[cfg(test)]` since it looks like the crate is the same and it would work to save the boilerplate.

Reply:
Done